prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Heather Merizan';
+    console.log('Value of fullName:', fullName);
+
+    var numberOfPets = '3';
+    console.log('Value of numberOfPets:', numberOfPets);
+
+    var likesVegetables = 'True';
+    console.log('Value of likesVegetables:', likesVegetables);
+
+    var nothing = 'null';
+    console.log('Value of nothing:', nothing);
```

Comment:
For numbers, `null`, and booleans we want to leave off the quotes so that the javascript engine doesn't interpret them as strings.